roles/manifest/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Download Manifest
+  import_tasks: download_manifest.yml
+  when: manifest_download == True
```

Comment:
`== True` seems unnecessary?

Is ansible-lint running on the roles? It should have yelled at that.